db/migrate/20191111104035_add_request_to_apps.rb

```diff
@@ -0,0 +1,5 @@
+class AddRequestToApps < ActiveRecord::Migration
+  def change
```

Comment:
Kindly use `up` and  `down` methods instead to make migration reversible.
Also instead of adding a column to Apps, kindly create a new table `AppEditRequests`
that has a foreign key to `Apps` with columns:

- `description` as text
- `features` as text
- `feedback` as text
- `approval_time` as datetime
- `approver` as a foreign key to a coach (ie. User table)

Reply:
The reason for creating a table instead of using a new column is so that we can log older requests in the database if the need ever sufficed.